recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -88,7 +88,7 @@ for %%i in ("%~dp0.") do set "SCRIPT_DIR=%%~fi"
 <cuda.version set /p CUDA_VERSION=
 del cuda.version
 if not "%CUDA_VERSION%" == "None" (
-    if "%CUDA_VERSION:~0,2%" == "12" (
+    if "%CUDA_VERSION:~0,2%" GEQ "12" (
```

Comment:
This will probably do stringly ordering. Generally, batch scripts are horrible with booleans, much less string ordering. It will probably work, but not excited about this.

Reply:
GEQ will do a numeric comparison where possible: https://ss64.com/nt/geq.html

I think this is safe, and yes, batch scripts are horrible.